apps/web/pages/org/[id]/[event]/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { NextPageWithLayout } from 'next';
+import { DashboardLayout } from "@app/layout";
+import React from 'react';
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableHeader,
+    TableRow,
+} from '@app/ui/components/table';
+import { AiOutlineDelete } from 'react-icons/ai'; 
+import{ FiEdit2 }from 'react-icons/fi';
```

Comment:
check if  Delete and edit icon is available in same package so that we can reduce the imports